package.json

```diff
@@ -309,6 +309,23 @@
             "markdownDescription": "**Experimental**: Run `swift build` in the background whenever a file is saved. It is possible the background compilation will already be running when you attempt a compile yourself, so this is disabled by default.",
             "order": 10
           },
+          "swift.backgroundIndexing": {
```

Comment:
we should put this under the "sourcekit-lsp" subsection like "swift.sourcekit-lsp.disable"